docs: add initial README and AUTHORS file

AAEmara opened the pull request:
Added the following documents for project documentation:
  - README file for poject documentation.
  - AUTHORS file for contributors recognition.

Paritally-Addresses #23

AAEmara committed:
docs: add initial README and AUTHORS file

Added the following documents for project documentation:
  - README file for poject documentation.
  - AUTHORS file for contributors recognition.

Paritally-Addresses #23

AhmedRamadanAR reviewed (commented):
The Documentation is great but there is one point missed : User is able to change Measurement unit .
Authors file :  I think two Authors are missing.

kerolosNabil247 reviewed (commented):
you made a great work in the  documentation
but the debouncing is to the manually search and also for the predefined dropdown list.
only the last search will be cached and appear imedietly

AAEmara committed:
docs: update AUTHORS files with contributors

Updated the AUTHORS file with new remaining contributors.

Closes #23

AAEmara committed:
docs: update features finished in README

Updated the list of features finished in README
file.

Closes #24

AAEmara commented:
> The Documentation is great but there is one point missed : User is able to change Measurement unit . Authors file : I think two Authors are missing.

  -  I updated the Finished Features in the README with the toggle weather measurement units feature (fahrenheit to celsius). 
  -  I updated the AUTHORS file with the missing contributors.

AAEmara commented:
> you made a great work in the documentation but the debouncing is to the manually search and also for the predefined dropdown list. only the last search will be cached and appear imedietly

  - I updated the previous features into the features list.
